Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -1553,7 +1553,7 @@ public void HandleAutoplayTabInput(GameTime gameTime)
             }
 
             // Only allow offset changes if the map hasn't started or if we're on a break
-            if (!IsSongSelectPreview && Ruleset.Screen.Timing.Time <= 5000 || Ruleset.Screen.EligibleToSkip)
+            if (!IsSongSelectPreview && !(this is EditScreen) && (Ruleset.Screen.Timing.Time <= 5000 || Ruleset.Screen.EligibleToSkip))
```

Comment:
I don't think this line makes sense?

This will always equal true because `this` will always be equal to`GameplayScreen`

Reply:
And this case should already be covered by the `!IsSongSelectPreview`?

Reply:
From my testing `!IsSongSelectPreview` did not cover it. 
And my fix did fix it from my testing, but maybe I missed something? 
Will recheck later.

Reply:
If it doesn't, then it should. Because ideally it would disable offset changes anywhere a map preview is active.